Cargo.toml

```diff
@@ -540,7 +539,7 @@ solana-sysvar = "=2.2.1"
 solana-sysvar-id = "=2.2.1"
 solana-test-validator = { path = "test-validator", version = "=2.2.6" }
 solana-thin-client = { path = "thin-client", version = "=2.2.6" }
-solana-transaction = "=2.2.1"
+solana-transaction = "=2.2.2"
```

Comment:
`solana-sdk` has a [strict patch level dependency](https://crates.io/crates/solana-sdk/2.2.2/dependencies) on v2.2.2 here so had to bump